components/list/index.html

```diff
@@ -0,0 +1,315 @@
+          <li><a href="/">Home</a></li>
+          <li class="active-page">
+            <a href="./documentation.html">Documentation</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="theme-toggle">
+        <input
+          type="checkbox"
+          class="theme-toggle__checkbox"
+          id="theme-toggle"
+        />
+        <label class="theme-toggle__label" for="theme-toggle">
+          <span> <i class="fas fa-moon"></i></span>
+          <span> <i class="fas fa-sun"></i></span>
+          <span class="ball"></span>
+        </label>
+      </div>
+    </header>
+
+    <section class="documentation-section">
+      <main class="documentation-section__main">
+       
+          <h1 class="title">List</h1>
+          <p class="paragraph">
+           
+          </p>
+          <h3 class="subtitle">Stacked Notification</h3>
+          <p class="paragraph"></p>
+          <div class="component-container">
+            <div class="component-container__actions">
+              <button
+                class="active component-container__actions__btn-live-demo"
+              >
+                Live Demo</button
+              ><button class="component-container__actions__btn-code-snippet">
+                Code Snippet
+              </button>
+            </div>
+            <div class="component-container__live-demo">
+              
+              <div class="notification">
+                <div class="avatar small">
+                  <img src="https://picsum.photos/536/354" alt="sample"/>
+                </div>
+
+                <div>
+                  <h5 class="notification-title">Mohit Yadav Wants to be Your Friend</h5>
+                  <p class="notification-details">
+                    <i class="fas fa-user-plus"></i> 25 min ago
+                  </p>
+                  
+                </div>
+               
+              </div>
+<!--  -->
+
+ <div class="notification">
+                <div class="avatar small">
+                  <img src="https://picsum.photos/536/354" alt="sample"/>
+                </div>
+
+                <div>
+                  <h5 class="notification-title">Mohit Yadav Liked Your Post</h5>
+                  <p class="notification-details">
+                    <i class="fas fa-heart"></i> 25 min ago
+                  </p>
+                  
+                </div>
+                
+              </div>
+<!--  -->
+
+<div class="notification">
+                <div class="avatar small">
+                  <img src="https://picsum.photos/536/354" alt="sample"/>
+                </div>
+
+                <div>
+                  <h5 class="notification-title">Mohit Yadav Commented on Your Post</h5>
+                  <p class="notification-details">
+                    <i class="fas fa-comment-alt"></i> 25 min ago
+                  </p>
+                  
+                </div>
+                
+              </div>
+
+             
+
+            
+
+              
```

Comment:
It would be easier to read by removing unnecessary spacing and beautifying the whole code.
Else looks good to merge.

Reply:
I don't know this formatting issue although I am using the prettier extension. But I'll try to fix that out. Thank for the review of this PR.